### PR TITLE
WorxLandroid Binding Readme.md - First step

### PR DESCRIPTION
**Pull request by sihui62**
see also PM in community forum

### CODE REVIEW

File: bundles/org.openhab.binding.worxlandroid/README.md [hunk 1]
```diff
@@ -1,858 +1,649 @@
...
 
-The binding retrieves properties from the API. To view the properties open the thing in PaperUI and click on 'SHOW PROPERTIES':
+##### common
 
-Bridge:
-<br>
-<img src="images/SC_Bridge.png" width="40%">
+| Channel          | Type       | ChannelName             | Values            |
+|------------------|------------|-------------------------|-------------------|
+| status           | `String`   | common#status           | *1 (see below)    |
```
Comment (clinique) [inline bundles/org.openhab.binding.worxlandroid/README.md]: Please, remove the "`" backticks

File: bundles/org.openhab.binding.worxlandroid/README.md [hunk 2]
```diff
@@ -1,858 +1,649 @@
...
+*2: Values for **status** Channel:
 
-##### common
+UNKNOWN, IDLE, HOME, START_SEQUENCE, LEAVING_HOME, FOLLOW_WIRE, SEARCHING_HOME, SEARCHING_WIRE, MOWING, LIFTED, TRAPPED, BLADE_BLOCKED, DEBUG, REMOTE_CONTROL,  GOING_HOME, ZONE_TRAINING, BORDER_CUT, SEARCHING_ZONE, PAUSE, MANUEL_STOP
+
+##### config
+
+| Channel   | Type       | ChannelName      |
+|-----------|------------|------------------|
+| timestamp | `DateTime` | config#timestamp |
```
Comment (clinique) [inline bundles/org.openhab.binding.worxlandroid/README.md]: Same here

File: bundles/org.openhab.binding.worxlandroid/README.md [hunk 3]
```diff
@@ -1,858 +1,649 @@
...
 
-```
-MowerBat_Chart*, MowerBatTemp_Chart*, MowerBatStatus_Chart* : strategy = everyMinute
-```
+##### sunday (Slot 1)
 
-### .things
+| Channel  | Type          | ChannelName     |
+|----------|---------------|-----------------|
+| enable   | `Switch`      | sunday#enable   |
```
Comment (clinique) [inline bundles/org.openhab.binding.worxlandroid/README.md]: Here also

**Review by clinique — CHANGES_REQUESTED** [review]
Thanks for your complete review of the documentation. Some minor observations.

**Comment by sihui62** [comment]
> Please, remove the "`" backticks

done.

**Review by clinique — APPROVED** [review]

**Pull request merged**